api/src/models/RecruiterProfile.ts

```diff
@@ -67,4 +67,10 @@ export class RecruiterProfile extends ModelBase<RecruiterProfile> {
     field: 'deleted_at',
   })
   deletedAt: Date
+
+  @AllowNull(true)
+  @Column({
+    type: DataType.STRING(255),
+  })
+  city: string;
```

Comment:
field is nullable in db but not nullable here

Reply:
Hi! Thank you for the review! 
Sorry, I thought decorator itself will do the thing, fixed it, thanks!